lib/coverband/collectors/coverage.rb

```diff
@@ -77,10 +75,6 @@ def filtered_files(new_results)
         end.select { |_file_name, coverage| coverage.any? { |value| value&.nonzero? } }
       end
 
-      def ready_to_report?
```

Comment:
I thought we still allowed this for resque polling? Not opposed to removing though

Reply:
if we remove this I think we can remove @reporting_frequency and the coverband config related to it

Reply:
Double checked and I already deleted from configuration.

> I thought we still allowed this for resque polling?

Doesn't look like we are using this within our resque integration as we were passing true for force_report. Was thinking about it and I feel like this configuration is confusing when used along with background reporting which reports every 30 seconds by default and has it's own `Configuration#background_reporting_sleep_seconds`. Perhaps we remove it for now and bring something like it back in another form if needed?

Reply:
yeah agreed and it is really an edge case and if we want to simplify we just have to make calls, on a pretty high traffic resque installation we have been fine at 100% coverage, it does slow down the background jobs but not much can be done about that with the forking model and the coverage was worth it for us.

Reply:
Ok sounds good.